### PR TITLE
Make it a react app

### PR DESCRIPTION
**Pull request by yossef-khaled**

### CODE REVIEW

**Comment by yossef-khaled** [comment]
There is a conflict, but I don't know why?

**Comment by mhashim6** [comment]
for #2

File: src/assets/compressed-logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...63-7.353-12.487-2.96-28.22-11.207-40.843-11.248-10.538-.038-22.88 5.812-33.319 7.899-23.27 4.653-47.218 8.002-70.937 8.583-24.237.594-49.385-1.682-73.087-6.89-11.406-2.505-23.99-8.783-35.469-9.686-7.264-.572-16.583 3.698-23.645 5.441-13.731 3.389-27.876 6.074-41.918 7.76-49.655 5.96-95.437 3.341-142.95-12.732m-24.72 37.619C84.39 103.87 74.345 170.788 74.345 223.819v180.567c0 64.2-.166 124.982 23.027 185.942 13.554 35.597 34.41 69.557 59.916 97.808 63.865 70.737 147.292 99.957 240.574 99.957 55.955 0 108.608-9.918 157.997-37.47 68.218-38.057 120.864-105.202 143.997-179.641 17.076-54.95 16.15-109.711 16.15-166.596V219.52c0-51.926-8.372-116.946 21.496-162.296-27.36.644-34.252 43.614-37.326 64.488-10.5 71.275-4.592 146.237-4.592 218.186 0 51.146 3.59 103.873-1.256 154.772-6.266 65.802-33.097 130.883-78.374 179.493-17.092 18.349-35.939 34.564-56.87 48.363-24.438 16.111-51.401 27.285-79.535 35.015-38.093 10.467-79.074 8.924-118.229 7.965-31.108-.763-62.953-9.946-91.359-22.103-24.629-10.54-47.167-25.373-67.712-42.457C140.91 649.942 103.543 573.61 96.025 494.67c-2.112-22.173-.166-45.434-.166-67.713v-134.35c0-59.865 5.158-122.647-6.45-181.643-3.764-19.13-11.289-52.633-35.468-53.74Z"/></svg>
```
Comment (mhashim6) [inline src/assets/compressed-logo.svg:1]: Lovely!

File: src/Components/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import logo  from '../../assets/compressed-logo.svg';
+
+function Home() {
+    return(
+        <div>
+            <div class="frame">
+                <img src={logo} alt=""/>
+              <p class="title">The Unhappy Folk</p>
+              <p class="subtitle">Timeless, Placeless Projects.</p>
```
Comment (mhashim6) [inline src/Components/Home/Home.jsx:9]: Please remove "projects".

**Review by mhashim6 — CHANGES_REQUESTED** [review]

**Comment by yossef-khaled** [comment]
What do you mean ?

**Comment by mhashim6** [comment]
> What do you mean by for #2 ?

Referencing the issue number to link with this PR.

**Review by mhashim6 — APPROVED** [review]

File: public/index.html [hunk 3]
```diff
@@ -45,14 +45,12 @@
       href="https://fonts.googleapis.com/css2?family=Gideon+Roman&display=swap"
       rel="stylesheet"
     />
-    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link rel="stylesheet" href="../src/index.css" type="text/css" />
```
Comment (mhashim6) [inline public/index.html:48]: please put index.css in the same folder.

**Review by mhashim6 — CHANGES_REQUESTED** [review]
please also remove logo.svg and rename compressed-logo.svg

**Comment by mhashim6** [comment]
@yossef-khaled please commit `yarn.lock`

**Comment by yossef-khaled** [comment]
I am not using yarn.

**Comment by yossef-khaled** [comment]
I was trying to pull your changes locally, but turned out with this last commit !!

**Comment by mhashim6** [comment]
> I am not using yarn.

You should!

**Comment by mhashim6** [comment]
> I was trying to pull your last commit locally, but turned out with this last commit !!

That's completely normal.

**Review by mhashim6 — APPROVED** [review]

File: public/index.html (hunk 3 above)
Comment (mhashim6) [reply public/index.html:52]: @yossef-khaled

**Comment by mhashim6** [comment]
@yossef-khaled please look at the last commit.

**Comment by yossef-khaled** [comment]
Really ??

**Comment by mhashim6** [comment]
A brilliant job! Merging.

**Pull request merged**